Add forest story

sjaakvandenberg opened the pull request:
Please review this, @casey-collab

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/udacity/create-your-own-adventure/1626)
<!-- Reviewable:end -->

casey-collab commented:
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

cbuckey-uda commented on english/forest/drag/drag.md:
```diff
@@ -0,0 +1,3 @@
+You drag yourself further into the woods. Along the way, you disturb a nest of rattle snakes. Before you can get away, they lunge at you and inject you with their venom. You die in even more pain than you were before.
+
+&mdash; By [@svdb](https://twitter.com/svdb)
```
Could you remove these attributions? The mdash is causing a bug in Casey when she tries to remove the pull request, and I'd rather not have every author put their twitter handle into the story. If people want to see who wrote this section, they can look at the commit history and find your GitHub profile.

sjaakvandenberg commented:
Removed attribution and mdashes. Thanks Caroline!

cbuckey-uda commented:
Thank you! @casey-collab, please take another look.

casey-collab commented:
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

cbuckey-uda commented:
Hmm, not sure what that is about. Hold on, I am looking into it.

cbuckey-uda commented on english/forest/scream/sing/sing.md:
```diff
@@ -0,0 +1,23 @@
+
+"What is it?" you replied.
+
+<scene cut>
```
It looks like this is the line that is causing Casey trouble. I should fix her, but you should also probably remove the angle brackets in this line, since it's getting interpreted as an html tag. When I view the file on GitHub, this line does not appear for that reason. (If you want, you could use asterisks to put this in italics - that is, use `*scene cut*` instead of `<scene cut>`.)

cbuckey-uda commented on english/forest/pockets/pockets.md:
```diff
@@ -0,0 +1,11 @@
+
+You light the flare and wave it over your head! Suddenly you realize, that under all this thick foliage, there is no way they are able to see it! What to do??!
+
+You think quickly. Of course! You throw the flare at the tree that holds your parachute. Immediately, the flare sets it ablaze! You hear the sounds of the helicopter's rotor blades approaching. It lands in a small nearby field. You're saved! As you limp towards the helicopter, and see stewardesses approach you with champagne and refreshments, you think to yourself:
```
This line and quite a few others are above the 120 character limit. This will make tools like `git diff` difficult to use. Could you wrap all the long lines? You don't need to add additional blank lines, just press enter somewhere in the middle of each line. You may also be able to get your text editor to do this automatically, depending on which one you use.

cbuckey-uda commented:
It looks to me like this story segment is not reachable from the existing story. That is, if a reader starts at `language.md` and keeps clicking links, they will never reach your addition. You can fix this by finding a page in the existing story that should lead to this change and adding a link to it.

If you all these things up, this will be good to merge. Sorry about the delayed comments due to Casey's bug.

sjaakvandenberg commented:
I've incorporated all your suggestions. I set a hard wrap to 80 characters, added a link in english/marshmellow.md and replaced the angle brackets with asterisks. Thanks for your help and patience!

sjaakvandenberg commented:
@casey-collab, please review! :)

casey-collab commented:
Unfortunately, pull requests adding more than 100 lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to add 100 lines or fewer.

sjaakvandenberg commented:
Cut several parts of the story to make it shorter to enable @casey-collab to automatically merge it.

casey-collab commented on english/forest/pockets/pockets.md:
```diff
@@ -0,0 +1,17 @@
+You light the flare and wave it over your head! Suddenly you realize, that
+under all this thick foliage, there is no way they are able to see it... Of
+course! You throw the flare at the tree that holds your parachute. Immediately,
+the flare sets it ablaze! You hear the sounds of the helicopter's rotor blades approaching. It lands in a small nearby field. You're saved! As you limp
```
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.

casey-collab commented on english/marshmallow.md:
```diff
@@ -15,7 +15,7 @@ Grave danger.
 *(Is there another kind?)*
 
 The type of mortal danger that so often accompanies confectionary inspired interior design.
-Quickly surveying your immediate surrounds, you notice a garishly dressed, 
```
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.

casey-collab replied:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.

casey-collab commented on english/marshmallow.md:
```diff
@@ -291,7 +294,7 @@ Do you:
 [Pull out your proton pack!](ghost-buster/ghost-busted.md)
 
 [You take the red pill](red-pill/red-pill.md)
- 
```
Here's another deleted line.

casey-collab replied:
Here's another deleted line.

casey-collab commented on english/forest/forest.md:
```diff
@@ -0,0 +1,24 @@
+An overwhelming pain echoes through your head. You feel the taste of dirt in
```
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

casey-collab commented:
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

sjaakvandenberg commented:
@casey-collab, please review

casey-collab commented:
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

sjaakvandenberg commented:
@casey-collab, please review

casey-collab commented:
Looks great! :star:
